client/web/web.go

```diff
@@ -657,6 +658,8 @@ func (s *Server) serveGetNodeData(w http.ResponseWriter, r *http.Request) {
 			return p == route
 		})
 	}
+	data.AdvertisingExitNodeApproved = routeApproved(exitNodeRouteV4) || routeApproved(exitNodeRouteV6)
```

Comment:
Actually, this might have to be an `&&` - @willnorris do you know

Reply:
In this case, I honestly don't think it matters.  For the purposes of advertising an exit node, they're always handled together.  I don't think it's possible to have one of them approved without the other.